experimental/traffic-portal/src/app/core/cdns/cdn-table/cdn-table.component.html

```diff
@@ -0,0 +1,29 @@
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<mat-card class="table-page-content">
```

Comment:
In a recent [PR](https://github.com/apache/trafficcontrol/pull/7582) I changed some classes around (would probably need to rebase to see them), this class no longer exists you should use `page-content` instead

Reply:
> In a recent [PR](https://github.com/apache/trafficcontrol/pull/7582) I changed some classes around (would probably need to rebase to see them), this class no longer exists you should use `page-content` instead

Using `page-content` instead in 54ce4b08ad